build/SkiaSharp.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <PackageReference Include="SkiaSharp" Version="1.68.0" />
-    <PackageReference Condition="'$(IncludeLinuxSkia)' == 'true'" Include="Avalonia.Skia.Linux.Natives" Version="1.68.0.2" />
+    <PackageReference Include="SkiaSharp" Version="1.68.1-rc.153" />
```

Comment:
@Gillibald  Just for reference, i wonder when can we expect a full release version of both HarfBuzzSharp and SkiaSharp?

Reply:
The plan is to have a release a month after the first preview. So if nothing breaks we get it until end of november.